docs/Build-Install-MRP-App.md

```diff
@@ -0,0 +1,205 @@
+#Build and Install MRP App 
+This document explains how to build and manually install the MRP Application
+
+To be able to run through this walkthrough there is a dependency on Java 8 SDK and JRE8. These can be found at the link below:
+http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
+
+##Build the MRP Solution 
+In order to build the MRP solution we will need to install some more dependencies. 
+1.  To build the project, navigate to the OrderService folder with cmd/PowerShell console and initiate the build by calling the gradlew file with the parameter of the build.gradle file.
+
+`PS> .\gradlew build`
+
+Once run you can find the compiled jar file under the “build/libs” folder.
+
+##Open the solution in Eclipse 
+
+We recommend Eclipse IDE but you can use whatever IDE you want.
+
+1. To be able to add the OrderService to Eclipse as a project first we need to add the plugin.
+2.  Open the file build.gradle.
+3.  Add the following line into the section of plugins.
+
+`apply pulgin: 'eclipse'`
+
+4. Run the following command in the PowerShell console
+
+`PS:>.\gradlew eclipse
+PS:>.\gradlew buildDependents`
+
+5.  Open Eclipse and create a new workspace.
+6.  Right click on the Project Explorer and select import a new project.
+7.  Browse to the OrderService root folder and select it
+8.  Check the OrderService project and click finish. 
+
+##Create a Linux VM 
+Create a Linux VM in Azure and signin onto it.
+
+##Install Dependencies
+1.  With you ssh console now open we need to add the following dependencies:
+• Java 8 SDK
+• Java 8 JRE
+• MongoDB
+2.  Enter the following command to install the dependencies above.
+`sudo apt-get install openjdk-8-jdk openjdk-8-jre mongodb`
+
+3.  To setup Java type the following commands to set your environment variables.
+`export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+export PATH=$PATH:/usr/lib/jvm/java-8-openjdk-amd64/bin`
+
+4.  To check your Java 8 installation, type “java -version”
+NOTE: if you are still referencing an older version of Java that was installed as part of your machine you can update this using the following command:
+
+`sudo update-alternatives --config java`
+You will be presented with a list of Java installations. Type the number you wish to set as your java default.
+
+##Setup MONGODB
```

Comment:
Setup Mongo and Setup webservice are to try and host/run your app, not create the builds.  Do we have these steps already defined in our existing deployment scripts/docs?  This doc should be purely spit out builds for web, order, integration services.  Then link to the deployment doc(s) which will spin up the Linux VM, install dependencies, etc.